src/storybook/components/related-components/descriptions/radio-button-description.jsx

```diff
@@ -0,0 +1,14 @@
+import { useMemo } from "react";
+import { RelatedComponent } from "../../related-component/related-component";
+import RadioButton from "../../../../components/RadioButton/RadioButton";
+
+export const RadioButtonDescription = () => {
+  const component = useMemo(() => <RadioButton text="Selection" checked={true} />, []);
```

Comment:
please use shorthand syntax for setting true boolean value for a prop `<RadioButton text="Selection" checked />`.